courses/fundamentals_of_ada/050_array_types.rst

```diff
@@ -675,86 +675,6 @@ Extra Object-Level Operations
 
    - Portion of array
 
----------
```

Comment:
Without this we never define what a slice is or how to specify it

Reply:
Good catch, I moved it up from "Ada2012 operations" (which was unrelated)